testing/web3signer_tests/src/lib.rs

```diff
@@ -10,15 +10,20 @@
 //! - The signatures generated by Web3Signer are identical to those which Lighthouse generates.
 //!
 //! There is a build script in this crate which obtains the latest version of Web3Signer and makes
```

Comment:
This comment is no longer accurate, could mention `get_web3signer` instead

Reply:
Good catch!

Reply:
Fixed